apps/music/js/metadata/flac.js

```diff
@@ -41,7 +43,16 @@ var FLACMetadata = (function() {
       if (!block) {
         return metadata;
       }
-      return readAllComments(block.view, metadata);
+      readAllComments(block.view, metadata);
+
+      LazyLoader.load('js/metadata/vorbis_picture.js', function() {
+        VorbisPictureComment.parsePictureComment(metadata).
+          then(function (metadata) {
+            // nothing to do, metadata is being modified anyway
```

Comment:
in the end, I'll remove FLAC support for cover art until I solve the other problem with. I'll keep these comments in mind.